cmake/ThriftGenerate.cmake

```diff
@@ -81,7 +81,7 @@ add_custom_command(
   COMMAND ${THRIFT1}
     --strict "--allow-neg-enum-vals"
     --templates ${THRIFT_TEMPLATES}
-    --gen "mstch_cpp2:include_prefix=${include_prefix},process_in_event_base,stack_arguments"
+    --gen "mstch_cpp2:include_prefix=${include_prefix},stack_arguments"
```

Comment:
After digging the usage about "process_in_event_base",  i am not sure  it is not correct for current nebula thread model.

Typically,  we don't want the service function call being invoked inside IOThreadPool,  right?  And By default,  thrift will use a ThreadManager(CpuThreadPool) to take care the real job.  So all logic inside nebula are based on this thread model.  For example,  in write path,  the real processing logic happen inside ThreadManager and for some read path not normally (For example, scan path etc )are reuse this pool too.

wdyt? @chenxu14

Reply:
Hi, thanks for your input, after tracing the code stack, here's what I found.
After remove “**process_in_event_base**”, the call stack will works like this (take getEdgeProps as an example)
```
StorageServiceAsyncProcessor::process(StorageService.cpp)
    —>apache::thrift::detail::ap::process(GeneratedCodeHelper.h)
        —> StorageServiceAsyncProcessor::getBinaryProtocolProcessMap(StorageService.cpp)
            —> process_pmap(GeneratedCodeHelper.h)
                   # get function pointer through Cpp2RequestContext#getMethodName and pmap
                   # the method signature will be _processInThread_getEdgeProps
                   —> StorageServiceAsyncProcessor::_processInThread_getEdgeProps(StorageService.tcc)
                       —> GeneratedAsyncProcessor::processInThread(AsyncProcessor.h)
                           —> ThreadManager#add(PriorityEventTask, 0, 0, true, true)
                                  # add Task to ThreadManager
```
So the service function will be invoked through **ThreadManager**, not **IOThreadPool**.
The core difference is that the task type add to ThreadManager are different. before remove “process_in_event_base”, task type is a FunctionRunner, has no priority info, so ThreadManager will treat all request as NORMAL, After remove “process_in_event_base”, task type will be PriorityEventTask which has priority info.

Reply:
Indeed, as the name "process_in_event_base" meaning,  if pass the flag into thrift compiler,  `ThreadManager` should be disabled, right?  
But as your description above,  no matter w/o the flag, the real job always been done inside ThreadManager.

That's weired, maybe a bug for thrift?

Reply:
There seems to be no official introduction how to use it, not easy to say if it's a bug.